shared/src/helpers/date/get-formatted-date/get-formatted-date.ts

```diff
@@ -18,6 +19,7 @@ const getFormattedDate = (date: string, formatDate: FormatDate): string => {
     case 'HH:mm, dd.MM':
     case 'yyyy-MM-dd':
     case 'dd-MM-yyyy':
+    case 'dd.MM.yyyy':
```

Comment:
We should use same format as on frontend

Reply:
## The same as on frontend:
<img src="https://gyazo.com/eef172c243e34a29234177fa0dbe8f2f.png" width="40%"> <img src="https://gyazo.com/c0d3797709dce56d70043fa1768bd740.png" width="40%">